### ISSUE TITLE
Wire/connection system

### ISSUE THREAD

**Issue by adam4813** [trillek-team/tec#34]
Drawing and updating power and data wires between components that have appropriate ports for said wires.

Depends on #37 attachment.

Clicking on an attachment point that is compatible begins a connection action. Clicking on another compatible attachment point ends the connection action and creates a connection. Canceling the connection action should end the connection action.

This "link" will need to be saved as a component for both entities and have an identifier that describes the target attachment point.

**Comment by adam4813** [comment]
No longer relevant, or needs to be revisited later
